tests/util/settings.test.util.js

```diff
@@ -65,9 +37,6 @@ module.exports = {
     dropAll: dropAll,
     setApplicationClosed: setApplicationClosed,
     setApplicationNotYetOpen: setApplicationNotYetOpen,
-    settingApplicationNotYetOpen: settingApplicationNotYetOpen,
-    settingApplicationOpen: settingApplicationOpen,
-    settingApplicationClosed: settingApplicationClosed,
-    settingConfirmClosed: settingConfirmClosed,
-    settingRemoteHackathon: settingRemoteHackathon
+    settingConfirmClosed: Constants.Settings.CONFIRM_CLOSED,
+    settingRemoteHackathon: Constants.Settings.REMOTE_HACKATHON
```

Comment:
Can we remove this export and just have whatever code referencing this to just import Constants instead to be consistent?

Reply:
I agree, since `Constants.Settings` is not used directly anywhere in this util script.

Reply:
Sorry for my previous comment. I re-read this script and `Constants.Settings` is used in lines 8, 14, 25. 

I also looked at the other `*.test.js` and `*.test.util.js` scripts, and it seems like we have previously been exporting entities from the `*.test.util.js` scripts. However, that was due to the need to create objects using the constants ([example](https://github.com/hackmcgill/hackerAPI/blob/dev/tests/util/hacker.test.util.js#L13)).

Since here we are using the constants directly, should I just import them into `settings.test.js`, or would it be better to follow the same structure and keep the export?

Reply:
@chenxuan-zhou Let's move it all to Constants. It's more consistent with the rest of the constant vars.